media/css/firefox/personal.less

```diff
@@ -0,0 +1,194 @@
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FiraSans Regular';
+    src: url('/media/fonts/FiraSans-Regular.woff') format('woff');
```

Comment:
It will load all of the variants and versions of the font whereas we can be more selective if we have the fonts inside bedrock.